docs/guides/nwaku/upgrade-instructions.md

```diff
@@ -0,0 +1,20 @@
+---
+title: Upgrade Instructions
+hide_table_of_contents: true
+sidebar_label: Upgrade Instructions
+---
+
+import { AccordionItem } from '@site/src/components/mdx'
+
+If you are currently using Nwaku, are running an old version and want to upgrade your node, please follow the below migration instructions for each release newer than your current running version
+
+<AccordionItem title="v0.31.0">
+Named sharding was deprecated in this version. This means that pubsub topics will only be supported if they comply with the static sharding format: <code>/waku/2/rs/&lt;CLUSTER_ID&gt;/&lt;SHARD_ID&gt;</code><br /><br />
+
+In order to migrate your existing application, you need to:
+
+1. Make sure that your clients are sending messages to pubsub topics in the required format.
```

Comment:
Please take a look now. The idea here was that if an user interacts with a node via REST clients or `js-waku`, that they use valid pubsub topics.

It's hard to get into details because there's so many ways of interacting with a node. But at the end it boils down to REST and `js-waku` usage. And when using any of them, valid topics should be configured.

Let me know if you have a better way of explaining it :))